task-sdk/src/airflow/sdk/log.py

```diff
@@ -243,6 +276,14 @@ def json_processor(logger: Any, method_name: Any, event_dict: EventDict) -> str:
         (
             dict_tracebacks,
             structlog.processors.UnicodeDecoder(),
+            structlog.processors.CallsiteParameterAdder(
+                {
+                    structlog.processors.CallsiteParameter.FUNC_NAME,
+                    structlog.processors.CallsiteParameter.LINENO,
+                    structlog.processors.CallsiteParameter.PATHNAME,
+                }
+            ),
+            custom_format,
```

Comment:
Sorry no - this is flat out not acceptable. We use structlog to a) get structured log fields, and b) to be more performant. Sending every single log message through a stdlib log formatter is not acceptable.